Contents/Sketch/styles.js

```diff
@@ -24,12 +24,16 @@ var applyStyles = function (newStyles, sharedStyles) {
 
     if(style.Shadow) {
       var shadow = sharedStyle.shadows().addNewStylePart();
-      shadow.color = MSColor.blackColor().colorWithAlpha(style.Shadowopacity);
-      shadowAttr = style.Shadow.split(',');
-      shadow.offsetX    = shadowAttr[0];
-      shadow.offsetY    = shadowAttr[1];
-      shadow.blurRadius = shadowAttr[2];
-      shadow.spread     = shadowAttr[3];
+      var shadowColor = MSColor.colorWithSVGString("#" + style.Shadowcolor);
+      shadow.color = shadowColor.colorWithAlpha(style.Shadowopacity);
+      setShadowAttribute(shadow, style.Shadow);
+    }
+
+    if(style.Innershadow) {
```

Comment:
What value are you expecting for `Innershadow` here? If boolean, we will probably need to convert from string in the google sheet.

Do you have a copy of the google sheet you are using for this so I can see how to format the master sheet?